package.json

```diff
@@ -1 +1,25 @@
-{"name":"cap-frontend","version":"0.1.0","private":true,"scripts":{"dev":"next dev","build":"next build","start":"next start","lint":"next lint"},"dependencies":{"eslint-config-prettier":"^8.5.0","eslint-plugin-jsx-a11y":"^6.6.1","eslint-plugin-prettier":"^4.2.1","next":"12.3.1","prettier":"^2.7.1","react":"18.2.0","react-dom":"18.2.0"},"devDependencies":{"eslint":"8.23.1","eslint-config-next":"^12.3.1"}}
\ No newline at end of file
+{
+    "name":"cap-frontend",
+    "version":"0.1.0",
+    "private":true,
+    "scripts":{
+        "dev":"next dev",
```

Comment:
Could you include the change in port so parafront runs in port 3001?

"dev": "next dev -p 3001",